Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/Visuals/ScaleHandles.cs

```diff
@@ -200,21 +194,52 @@ protected Quaternion GetRotationRealignment(int handleIndex)
             Vector3 neutralHandle = signVector(HandlePositions[6]);
             Vector3 handlePos = signVector(HandlePositions[handleIndex]);
 
-            // Flip the handle if it's on the underside of the bounds.
-            Quaternion flip = Quaternion.Euler(0, 0, handlePos.y > 0 ? 0 : 90);
+            if(isFlattened)
+            {
+                Vector3 axis = Vector3.forward;
+                switch(currentFlattenAxis)
+                {
+                    case FlattenModeType.FlattenAuto:
+                        Debug.LogError("ScaleHandles should never receive FlattenAuto. BoundsControl should pass actualFlattenAxis");
```

Comment:
ActualFlattenAxis instead of actualFlattenAxis?

Reply:
Resolved